test/test_faker_company.rb

```diff
@@ -14,6 +14,10 @@ def test_duns_number
   end
 
   def test_logo
-	  assert @tester.logo.match(%r{http://pigment.github.io/fake-logos/logos/medium/color/\d+\.png})
+    assert @tester.logo.match(%r{http://pigment.github.io/fake-logos/logos/medium/color/\d+\.png})
+  end
+
+  def test_buzzword
+    assert @tester.buzzword.match(/\w+\.?/)
```

Comment:
We need to come up with a new policy on tests quickly. This test, while definitely following established Faker patterns, is practically useless in terms of actually testing that `#buzzword` works.